blocks/header/header.js

```diff
@@ -326,6 +326,28 @@ async function decorateSearch(block) {
   }
 }
 
+/**
+ * decorate Language to include flag image in href
+ * @param navSections
+ */
+function decorateLanguageNav(navSections) {
```

Comment:
Instead of selecting picture elements first and then looking for parent and child elements, it would be more efficient to select the parent li elements directly and handle their child elements in each iteration.

```
function decorateLanguageNav(navSections) {
   const listItems = navSections.querySelectorAll('.nav-drop > ul > li');

   listItems.forEach((li) => {
     const picture = li.querySelector('picture');
     const a = li.querySelector('a');

     if (picture && a) {
       const href = a.getAttribute('href');
       const txt = a.innerHTML;
       
       li.innerHTML = '';
       const newA = document.createElement('a');
       
       newA.setAttribute('href', href);
       newA.appendChild(picture);
       newA.appendChild(document.createTextNode(txt));
       
       li.appendChild(newA);
     }
   });
}
```